elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/metadata/SQLReferenceTable.java

```diff
@@ -122,6 +144,7 @@ public void resolveAndStoreAllReferencesAndJoins(Queryable queryable) {
         //References and joins are stored by their source that produces them (rather than the query that asks for them).
         Queryable key = queryable.getSource();
         SQLDialect dialect = queryable.getSource().getConnectionDetails().getDialect();
+        boolean isNested = queryable.isNested();
```

Comment:
This method is getting pretty long.  We might want to break it up.

Reply:
👍